src/main/scala/edu/ie3/simona/model/participant/SystemParticipant.scala

```diff
@@ -105,11 +105,7 @@ abstract class SystemParticipant[CD <: CalcRelevantData](
   def activeToReactivePowerFunc(
       nodalVoltage: ComparableQuantity[Dimensionless]
   ): ComparableQuantity[Power] => ComparableQuantity[Power] =
-    qControl.activeToReactivePowerFunc(
-      sRated.multiply(scalingFactor),
-      cosPhiRated,
-      nodalVoltage
-    )
+    qControl.activeToReactivePowerFunc(sRated, cosPhiRated, nodalVoltage)
```

Comment:
Why is the scaling factor removed here? Looks like something that belongs to #82, but not necessarily this PR?

Reply:
Good catch, think also something got mixed here... have been rolled back now.